tests/nightly/test_large_array.py

```diff
@@ -1168,6 +1169,74 @@ def check_correctness(mxnet_op, numpy_op, atol=1e-3):
     check_binary_broadcast()
 
 
+def test_linalg():
+    def batchify(mat):
+        return nd.array([mat.asnumpy()])
+
+    def check_diag(mat, val):
+        for i in range(LARGE_SQ_X):
+            assert mat[i, i] == val
+
+    def check_batch_diag(mat, val):
+        check_diag(mat[0], val)
+
+    def run_det(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = det(inp)
+        return inp.grad, out
+
+    def run_inverse(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = inverse(inp)
+        return inp.grad, out
+
+    def run_trmm(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = trmm(inp, inp)
+        return inp.grad, out
+
+    def run_trsm(inp):
+        inp.attach_grad()
+        with mx.autograd.record():
+            out = trsm(inp, inp)
+        return inp.grad, out
+
+    A = get_large_identity_mat(LARGE_SQ_X)
```

Comment:
You need to import it in this file now, since you have moved it to test_utils.py

Reply:
done